docs/stdlib/numbers.rst

```diff
@@ -207,39 +210,48 @@ from :eql:type:`str` and :eql:type:`json`.
 
     :index: numeric bigint
 
-    Arbitrary precision integer.
+    Represents an arbitrary precision integer.
 
-    The EdgeDB philosophy is that using bigint type should be an
-    explicit opt-in, but once used, the values should not be
-    accidentally cast into a numeric type with less precision.
+    In EdgeDB, the philosophy of :eql:type:`bigint` is that using this type
+    should be an explicit opt-in and not implicit. Once used, these values
+    should not be accidentally casted to a different numerical type that gives
+    less precision.
```

Comment:
```suggestion
    Our philosophy is that use of :eql:type:`bigint` should be an explicit
    opt-in and never implicit. Once used, these values should not be
    accidentally cast to a different numerical type that would lead to a loss of
    precision.
```